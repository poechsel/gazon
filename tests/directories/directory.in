login u1
pass p1
ls
cd dir1
mkdir dir1
cd dir1
ls
mkdir dir2
ls
rm dir2
ls
cd ..
ls
