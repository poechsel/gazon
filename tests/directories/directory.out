total 0
cd: dir1: No such file or directory
total 0
total 4
drwxr<*>xr-x 2 <*> <*> 4096 <*> dir2
total 0
total 4
drwxr<*>xr-x 2 <*> <*> 4096 <*> dir1
